apps/nuxt3-ssr/package.json

```diff
@@ -28,7 +28,7 @@
     "autoprefixer": "10.4.15",
     "graphql-tag": "2.12.6",
     "jsdom": "21.1.2",
-    "nuxt": "3.6.5",
+    "nuxt": "^3.7.4",
```

Comment:
renovate doesn't like ^ because then the actual build version is not deterministic. Did you do this with a reason?

Reply:
nuxt has its own preferred update system ( call) it uses the default ( with the ^ ) , ill change it

Reply:
i'll wait for ci build and check the preview before merging